lib/diagnostic.rb

```diff
@@ -108,10 +108,10 @@
 # Use shorthand to make this one line long.
 # Write your response as a Ruby string.
 
-Diagnostic.person_hash = 'your answer here'
+Diagnostic.person_hash = { favorite_number: 7, first_name: 'Jon', occupation: 'exterminator' }
 
 # Set a default return value to your `person` hash that concatenates the value
 # of person[:first_name] and a message about them.
 # Write your response as a Ruby string.
 
-Diagnostic.hash_default = 'your answer here'
+Diagnostic.hash_default = 'Hello #{person[:first_name]}'
```

Comment:
We can set a default return value like this!
`person.default = "#{person[:first_name]} is great"`
Good job on accessing the name correctly!